addon/components/item-chooser/create-item/component.js

```diff
@@ -0,0 +1,55 @@
+import Ember from 'ember';
+import ExportEntity from 'ember-upf-utils/export-entity/model';
+import layout from './template';
+
+const {
+  Component,
+  computed,
+  inject,
+  isBlank
+} = Ember;
+
+export default Component.extend({
+  layout,
+
+  store: inject.service(),
+
+  classNames: ['ember-basic-dropdown-content__create-item'],
+
+  formattedCreateButtonText: computed('searchTerm', function() {
+    return this.get('createButtonTextTemplate').replace(
+      '#item#',
+      this.get('searchTerm')
+    );
+  }),
+
+  actions: {
+    createItem(_, defer) {
+      if (isBlank(this.get('searchTerm'))) {
+        defer.resolve();
+        return;
+      }
+
+      let item = null;
+      if (this.get('recordTypeIsModel')) {
+        item = this.get('store').createRecord(this.get('recordType'), {
+          name: this.get('searchTerm')
+        });
+      } else {
+        item = ExportEntity.create({ name: this.get('searchTerm') });
```

Comment:
`ExportEntity` are you sure of that ?

Reply:
yes, why ?

Reply:
we were using the same system before with ExportEntity EmberObject to encapsulate the entities

Reply:
Good point :+1:

Reply:
😄